packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```diff
@@ -15,6 +15,7 @@
 
 	.components-modal__content {
 		padding-top: 0;
+		margin-bottom: 70px; // This should match the height of the footer
```

Comment:
How about using a variable here, defined at the top of the file, so it doesn't need to be manually kept in sync?

Reply:
Yes, nice idea! I did think about this but doubted myself 😅 Done in https://github.com/WordPress/gutenberg/pull/60641/commits/e5a985f7d5ea6eaaff3a8c47a6edb91de4f35f70.

Reply:
It is not guaranteed that the footer height will be exactly 70px. This may be because the fonts differ slightly depending on the OS. In my environment (Windows11, Chrome), the height is 68.8px.

![image](https://github.com/WordPress/gutenberg/assets/54422211/0e0e7a96-a895-42b4-bb3f-b683a8e5e294)

If we expect the footer to have a height of 70px, I think we need to give an explicit height to the footer itself.

My other idea is to apply `position:sticky`. This approach is also used in [the data view footer](https://github.com/WordPress/gutenberg/blob/e1842bb8dadd224ccf2e7499bcff781afb9ce499/packages/dataviews/src/style.scss#L43-L45). Styles will probably look like this:

```scss
.font-library-modal__tabpanel-layout__footer {
	margin: 0 #{$grid-unit-40 * -1} #{$grid-unit-40 * -1};
	position: sticky;
	bottom: #{$grid-unit-40 * -1};
	width: auto;
}
```

This will also fix the issue where the scrollbar is hidden behind the footer.

![scrollbar-overflow](https://github.com/WordPress/gutenberg/assets/54422211/2a158e9b-9a41-4b6b-a05e-b7e5d3da69c7)

What do you think?

Reply:
>If we expect the footer to have a height of 70px, I think we need to give an explicit height to the footer itself.

We're setting the height of the footer via the class `.font-library-modal__tabpanel-layout__footer` on line 48 of style.scss. In my testing, even when changing the font size, the height remains at 70px.

<img width="956" alt="image" src="https://github.com/WordPress/gutenberg/assets/1645628/41e9fc36-3a5d-4635-8e9b-67aa7dc060d0">

I'm not sure why you're seeing 68.8px there, although I know that's what the height was before I set it to 70px. Could this maybe be caused by a caching issue on the CSS?